parallel-consumer-core/src/main/java/io/confluent/csid/utils/Range.java

```diff
@@ -57,23 +76,17 @@ public void remove() {
         };
     }
 
-    public List<Integer> list() {
-        ArrayList<Integer> integers = new ArrayList<Integer>();
-        forEach(integers::add);
+    /**
+     * Potentially slow, but useful for tests
+     */
+    public List<Integer> listAsIntegers() {
+        ArrayList<Integer> integers = new ArrayList<>();
```

Comment:
```suggestion
        List<Integer> integers = new ArrayList<>();
```
Just catches eye at a glance...